backend/app/jobs/translations/translate_project_job.rb

```diff
@@ -0,0 +1,16 @@
+module Translations
+  class TranslateProjectJob < ApplicationJob
+    queue_as :default
+
+    rescue_from(ActiveRecord::RecordNotFound) do |exception|
```

Comment:
maybe should be moved to some base class for Translations or even to ApplicationJob

Reply:
yes, I would go for a superclass for translation jobs